### DISCUSSION TITLE
ibax network regional field managers should be allocated

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#1996] [Bug Submission]
According to the countries where the ibax network is concentrated, there should be social field teams such as Turkey, Singapore, amrekia, and these field teams should deal with the social problems and demands of their location. certain jurisdictions or responsibilities should also be possible. He should quickly resolve the social and trust problems of the region he is responsible for, according to his own jurisdiction. and allocate you a detailed report. Of course, these field teams can receive certain salaries or payouts in ibxc coin. @ibaxio @BadAssKyle @yolandadadada @ysfcndgr @akibax

**Comment by yolandadadada** [comment]
Already have it!

**Comment by Akrepkapo** [comment]
Yok bizimle iletişime geçin istersen
